### DISCUSSION TITLE
v1.4.9 Commit?

### DISCUSSION THREAD

**Discussion by abenson** [QtExcel/QXlsx#348] [General]
Was Qxlsx v1.4.9 intended to be the commit from two weeks ago? Because currently it does not contain the compilation fixes committed yesterday.

**Comment by j2doll** [comment]
Dear @abenson 
What I committed yesterday has not been tested yet. 
I will release additional versions in the future.
Thank you. 😃

**Reply by abenson** [reply]
In that case, v1.4.9 fails to build.
